storage/ndb/src/kernel/vm/mt.cpp

```diff
@@ -4004,37 +4022,34 @@ thr_send_threads::handle_send_trp(TrpId trp_id,
   assert(m_trp_state[trp_id].m_thr_no_sender == NO_OWNER_THREAD);
   if (m_trp_state[trp_id].m_micros_delayed > 0)     // Trp send is delayed
   {
+    if (!is_send_thread(thr_no))
```

Comment:
[Just for noting]
I think the root cause of adding this check is get_trp() could return a delayed trp, should we add a check in get_trp() instead of here?
Maybe we can discuss it on Monday.

Reply:
I need the distinction between having a delayed trp ready and no trp ready, so should stay as is